HelpMyStreetFE/HelpMyStreetFE/Startup.cs

```diff
@@ -218,6 +218,17 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                     name: "ForgottenPassword",
                     pattern: "forgotten-password",
                     defaults: new { controller = "Home", action = "ForgottenPassword" });
+
+                endpoints.MapControllerRoute(
+                 name: "OpenRequest",
```

Comment:
Does it matter that OpenRequest is singular here?

Reply:
i dont think so, but i've changed it to match anyway